diesel/src/migrations/mod.rs

```diff
@@ -132,38 +132,37 @@ pub fn mark_migrations_in_directory<Conn>(conn: &Conn, migrations_dir: &Path)
 ///
 /// See the [module level documentation](index.html) for information on how migrations should be
 /// structured, and where Diesel will look for them by default.
-pub fn revert_latest_migration<Conn>(conn: &Conn) -> Result<String, RunMigrationsError> where
+pub fn revert_latest_migration<Conn>(conn: &Conn, migrations_dir: &Path) -> Result<String, RunMigrationsError> where
```

Comment:
FTR, my "is this public API?" test: copy fn name, go to <http://docs.diesel.rs>, search for fn name, #results > 0 == public api